documentation/staging/content/userguide/managing-operators/installation.md

```diff
@@ -208,6 +219,13 @@ any running WebLogic Server instances created by the original operator. It
 is not necessary and such instances will continue to run without interruption
 during the upgrade.
 
+When you upgrade a 3.x operator to 4.0, it will also create a 
+[WebLogic Domain resource conversion webhook]({{< relref "/userguide/managing-operators/conversion-webhook.md" >}})
+deployment and its associated resources in the same namespace. If the conversion webhook deployment already exists in
+ some other namespace, then a new conversion webhook deployment is not created.
+The [WebLogic Domain resource conversion webhook]({{< relref "/userguide/managing-operators/conversion-webhook.md" >}})
+ automatically and transparently upgrades the existing Domains from the 3.x schema to the 4.0 schema.
+
```

Comment:
Suggest rewording and limiting the links so that readers are guided exactly where you want them. Something like:

When you upgrade a 3.x operator to 4.0, it will also create a WebLogic Domain resource conversion webhook deployment and its associated resources in the same namespace. If the conversion webhook deployment already exists in some other namespace, then a new conversion webhook deployment is not created.
The webhook automatically and transparently upgrades the existing Domains from the 3.x schema to the 4.0 schema. For more information, see [WebLogic Domain resource conversion webhook]({{< relref "/userguide/managing-operators/conversion-webhook.md" >}}).